electron/preload.js

```diff
@@ -0,0 +1,49 @@
+const { contextBridge, ipcRenderer } = require('electron')
+
+/**
+   * This file bridges the main process, electron.js, with the render process, Login.tsx,
+   * by exposing specific API using contextBridge.
+   */
+console.log('context bridge RULES!');
+
+const exposedAPI = {
+  /**
+   * @remarks
+   * Invoked by Login button to configure the user's local files
```

Comment:
Fantastic use of comments here. This function is very unclear, great to have context.